AppiumForMac/Server/Controller/AfMElementLocator.m

```diff
@@ -256,6 +259,8 @@ -(void)findAllUsingBaseUIElement:(PFUIElement*)baseUIElement results:(NSMutableA
             [self findAllUsingBaseUIElement:childElement results:results statusCode:statusCode];
         }
     }
+    *statusCode = kAfMStatusCodeSuccess;
```

Comment:
I'd rather set this at the beginning of the method

Reply:
Ok